### PR TITLE
Correct License banner instructions. 

### PR DESCRIPTION
**Pull request by kelvinalfaro**
Instructions read "Accept our license by changing the .env variable NEXT_PUBLIC_LICENSE_CONSENT to I agree"
NEXT_PUBLIC_LICENSE_CONSENT variable is set to only accept "agree" - removed the I from "I agree" in the banners instructions.

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]

**Comment by PeerRich** [comment]
great catch! we first had "I agree" but were a bit cautious of whitespace in .env files as some providers seem to have trouble with it :)

File: public/static/locales/en/common.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "new-event-type-btn": "New event type"
+  "new-event-type-btn": "New Event Type"
```
Comment (emrysal) [inline public/static/locales/en/common.json:2]: I believe this to be an accidental change?

Comment (PeerRich) [reply public/static/locales/en/common.json:2]: i actually dont mind the change

Comment (emrysal) [reply public/static/locales/en/common.json:2]: image
image

All of our buttons currently use "Lowercase continuing words" vs. "Camelcase Wording" - to my subjective eye "New event type" looks better than "New Event Type" but the most important thing here is consistency.

**Pull request merged**